oioioi/pa/controllers.py

```diff
@@ -61,10 +61,18 @@ def visible_contests_query(self, request):
     def can_register(self, request):
         return super().is_registration_open(request)
 
+    def get_registration_status(self, request):
+        return super().registration_status(request)
+
     def can_unregister(self, request, participant):
         return False
 
     def registration_view(self, request):
+
```

Comment:
looks like code duplication, find a way to remove it or at least add a TODO comment that this should be refactored.